ui/src/components/home/HomeContainer.tsx

```diff
@@ -0,0 +1,20 @@
+import { Box, Typography } from '@mui/material';
+import { FC, ReactElement } from 'react';
+
+const HomeContainer: FC<any> = (): ReactElement => {
```

Comment:
In the future, how are we going to define these types ya? esp the props for each component. Should we put them all into one type folder or have a type.ts in every component folder?

Reply:
ooo good question! i just googled and this is what i got: 
- https://stackoverflow.com/questions/56947690/react-component-type-in-typescript
- https://dmitripavlutin.com/typescript-react-components/
so i guess i should just remove the `<any>` here for now

Reply:
Ahh okok. Where do we wanna define the props type ya? Inside the current file / inside the component file but made a new seperate `type.ts` / made a ehole new folder `types` and store all the props type ther?

Thinking how to structure the whole ui for when we are gonna hav lots of components :")